src/Makefile

```diff
@@ -1,95 +0,0 @@
-default: test-exhaust bench
-
-DEFINES := -DHAVE_CLZ -O2 -g0
```

Comment:
`HAVE_CLZ` is not covered by the PR

Reply:
Thanks for catching that! Fixed.

Reply:
Don't CLZ test with zero, it's undefined behaviour.  :)

Reply:
Heh, thanks. Fixed.